lib/Doctrine/Common/DataFixtures/DependentFixtureInterface.php

```diff
@@ -13,7 +13,7 @@ interface DependentFixtureInterface
      * This method must return an array of fixtures classes
      * on which the implementing class depends on
      *
-     * @return array
+     * @return class-string[]
```

Comment:
I think this should be `@return FixtureInterface[]`

Reply:
I thought so too, but it turns out it expects an array of FQCN class names to do the sorting. Not the actual instances.

Reply:
https://github.com/doctrine/data-fixtures/blob/608a35a3b5bcc4214d116603095f8b0c51091592/lib/Doctrine/Common/DataFixtures/Loader.php#L170-L176

Reply:
So, it should be `@return \Doctrine\Common\DataFixtures\DependentFixtureInterface[]|\Doctrine\Common\DataFixtures\FixtureInterface[]`

Reply:
That's a bit of mess, I really don't get why other interfaces are not extending `FixtureInterface`... :disappointed:

Reply:
It should not return instances of that type, it should return FQCN (strings) of fixtures to load first.

Reply:
ah, you're right. My bad